### DISCUSSION TITLE
Using If...Then...Else

### DISCUSSION THREAD

**Discussion by dkekesi** [bijington/expressive#124] [Q&A]
Is there any documentation on how to use If...Then...Else in Expressive? The Wiki doesn't seem to have information about this. I need to return different results depending on evaluating part of a string.

**Comment by dkekesi** [comment]
Found it. Usage for future reference.
_If_ takes 3 parameters:
`If (condition that evaluates to bool, return value if condition is true, return value if condition is false)`
The word _If_ (like all built-in tokens) is case sensitive.

**Reply by bijington** [reply]
Apologies for the delayed response. I currently have a gap in the documentation on the wiki which I really need to sort out. 

As for the case sensitivity you can change this when building an expression ready for evaluation as covered here https://github.com/bijington/expressive/wiki/Usage#custom-options
